src/components/calculator.js

```diff
@@ -1,65 +1,63 @@
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="AC">AC</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button">+/-</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="%">%</button>
-              <button onClick={this.operationHandler} className="col-3 bg-warning border" type="button" value="÷">÷</button>
-            </div>
-            <div className="row justify-content-center align-content-center">
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="7">7</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="8">8</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="9">9</button>
-              <button onClick={this.operationHandler} className="col-3 bg-warning border" type="button" value="x">x</button>
-            </div>
-            <div className="row justify-content-center align-content-center">
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="4">4</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="5">5</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="6">6</button>
-              <button onClick={this.operationHandler} className="col-3 bg-warning border" type="button" value="-">-</button>
-            </div>
-            <div className="row justify-content-center align-content-center">
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="1">1</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="2">2</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value="3">3</button>
-              <button onClick={this.operationHandler} className="col-3 bg-warning border" type="button" value="+">+</button>
-            </div>
-            <div className="row justify-content-center align-content-center">
-              <button onClick={this.operationHandler} className="col-6 border" type="button" value="0">0</button>
-              <button onClick={this.operationHandler} className="col-3 border" type="button" value=".">.</button>
-              <button onClick={this.operationHandler} className="col-3 bg-warning border" type="button" value="=">=</button>
-            </div>
-          </div>
+  const { total, next, operation } = state;
+  return (
+    <div className="row justify-content-center">
+      <div className="col-sm-12 col-md-4" id="calc-container">
+        <div className="row justify-content-end px-1 bg-secondary">{total} {operation} {next}</div>
```

Comment:
- [ ] Sometimes the calculator displays two numbers on the screen, other times the display disappears. I think we should try to fix this behavior as is not intended.

![todo_list_no_screen](https://user-images.githubusercontent.com/79114233/144714550-1fdaf834-aa97-43aa-b935-0e3680e242c7.gif)